metricbeat/module/vsphere/host/_meta/data.json

```diff
@@ -15,6 +15,36 @@
     },
     "vsphere": {
         "host": {
+            "triggerd_alarms": [
+                {
+                  "name": "Host hardware system board status",
+                  "id": "alarm-121.host-12",
+                  "status": "red",
+                  "time": "2024-09-09T13:23:00.786759Z",
+                  "description": "Default alarm to monitor system boards.  See the host's Hardware Status tab for more details."
+                },
+                {
+                  "id": "alarm-124.host-12",
+                  "status": "red",
+                  "time": "2024-09-09T13:23:00.786773Z",
+                  "description": "Default alarm to monitor storage.  See the host's Hardware Status tab for more details.",
+                  "name": "Host storage status"
+                },
+                {
+                  "name": "Host memory usage",
+                  "id": "alarm-4.host-12",
+                  "status": "yellow",
```

Comment:
We should list only the critical alarms, yellow ones should not be listed.

Reply:
We can filleter out yellow ones by using pipeline in kibana. we have kept the yellow ones as well so that in future we don't have to update vshphere metricsets to add `yellow` alarms if customer requires it.